com.liferay.poshi.ide.core/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,27 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %bundle.name
+Bundle-SymbolicName: com.liferay.poshi.ide.core;singleton:=true
+Bundle-Version: 1.0.0.qualifier
+Bundle-Activator: com.liferay.poshi.ide.core.PoshiCore
+Bundle-Vendor: %bundle.vendor
+Require-Bundle: org.eclipse.ant.core,
+ org.eclipse.core.resources;visibility:=reexport,
+ org.eclipse.core.runtime;visibility:=reexport,
+ org.eclipse.debug.core,
+ org.eclipse.jdt.core,
```

Comment:
Do we need all of these dependencies on jdt.core, etc?  Can you make sure this list is the minimum list.